test/controllers/update/no.change.registrable.beneficial.owner.controller.spec.ts

```diff
@@ -98,24 +110,25 @@ describe("No change registrable beneficial owner", () => {
   });
 
   describe("POST tests", () => {
-    test(`Test redirect when 'no reasonable cause' is selected`, async () => {
+    test(`Test redirect to ${UPDATE_DO_YOU_WANT_TO_MAKE_OE_CHANGE_PAGE} page when 'no reasonable cause' is selected`, async () => {
       mockGetApplicationData.mockReturnValueOnce({ ...APPLICATION_DATA_MOCK });
       const resp = await request(app)
         .post(UPDATE_NO_CHANGE_REGISTRABLE_BENEFICIAL_OWNER_URL)
         .send({ [RegistrableBeneficialOwnerKey]: "1" });
 
       expect(resp.status).toEqual(302);
+      expect(resp.header.location).toEqual(UPDATE_DO_YOU_WANT_TO_MAKE_OE_CHANGE_URL);
       expect(mockSetExtraData).toHaveBeenCalledTimes(1);
     });
 
-    test(`redirects to the ${UPDATE_DO_YOU_WANT_TO_MAKE_OE_CHANGE_PAGE} page when 'has reasonable cause' is selected`, async () => {
+    test(`redirects to the ${UPDATE_REVIEW_STATEMENT_PAGE} page when 'has reasonable cause' is selected`, async () => {
       mockGetApplicationData.mockReturnValueOnce({ ...APPLICATION_DATA_MOCK });
       const resp = await request(app)
         .post(UPDATE_NO_CHANGE_REGISTRABLE_BENEFICIAL_OWNER_URL)
-        .send({ [RegistrableBeneficialOwnerKey]: yesNoResponse.No });
+        .send({ [RegistrableBeneficialOwnerKey]: "0" });
```

Comment:
Is "0" not the same as `yesNoResponse.No`?

Reply:
No it is not, yesNoResponse.No is enum resolving to 0, Ui submission is a string "0" or "1"